libraries/rib-compose/src/main/java/com/badoo/ribs/compose/ComposeTarget.kt

```diff
@@ -0,0 +1,41 @@
+package com.badoo.ribs.compose
+
+import android.content.Context
+import android.view.View
+import android.view.ViewGroup
+import android.widget.FrameLayout
+import androidx.compose.runtime.MutableState
+import androidx.compose.ui.viewinterop.AndroidView
+
+class ComposeTarget(
```

Comment:
It is hard to understand what `ComposeTarget` does because of naming.
Whaat do you think about `ChildComposeHost` or `ChildComposeAndroidViewHost`?